filestore/fs.py

```diff
@@ -153,16 +182,37 @@ def _db(self):
         if self.__db is None:
             conn = self._connection
             self.__db = conn.get_database(self.config['database'])
+            sentinel = self.__db.get_collection('sentinel')
+            versioned_collection = ['resource', 'datum']
+            for col_name in versioned_collection:
+                val = sentinel.find_one({'collection': col_name})
+                if val is None:
+                    raise RuntimeError('there is now version sentinel for '
+                                       'the {} collection'.format(col_name))
+                if val['version'] != self.version:
```

Comment:
Does it make sense to version the collections separately if there is only one matching version in FS? Maybe just one version for the whole database would do.